examples/gallery/ui/pages/controls_page.slint

```diff
@@ -106,16 +106,29 @@ export component ControlsPage inherits Page {
     }
 
     GroupBox {
-        title: @tr("ProgressIndicator");
+        title: @tr("ProgressIndicator | Spinner");
         vertical-stretch: 0;
 
         VerticalLayout {
             spacing: 16px;
 
-            i-progress-indicator := ProgressIndicator {
-                min-width: 160px;
-                progress: (i-slider.value - i-slider.minimum) / (i-slider.maximum - i-slider.minimum);
-                indeterminate: true;
+            HorizontalBox {
```

Comment:
Olivier and I discussed and want to propose the following alternative.

Before:
<img width="509" alt="Screenshot 2023-11-07 at 13 45 01" src="https://github.com/slint-ui/slint/assets/1486/438b2060-1973-4a54-b3d7-c43b233e21a4">


After:


<img width="509" alt="Screenshot 2023-11-07 at 13 45 17" src="https://github.com/slint-ui/slint/assets/1486/75560ad2-38a9-432a-9337-510d1930e76e">

This is what's in the GroupBox:

```
        GridBox {
            spacing: 16px;

            i-progress-indicator := ProgressIndicator {
                row: 0;
                col: 0;
                min-width: 160px;
                progress: (i-slider.value - i-slider.minimum) / (i-slider.maximum - i-slider.minimum);
                indeterminate: true;
            }

            Rectangle {
                row: 0;
                col: 1;
                rowspan: 2;
                Spinner {
                    progress: i-progress-indicator.progress;
                    indeterminate: i-progress-indicator.indeterminate;
                }
            }

            CheckBox {
                row: 1;
                col: 0;
                text: @tr("indeterminate");
                checked <=> i-progress-indicator.indeterminate;
            }
        }
```

Reply:
agreed and changed, thx.